procedures/issues.md

```diff
@@ -0,0 +1,50 @@
+# Issue tracking
+
+The issue tracker for the Leadership Council is located at <https://github.com/rust-lang/leadership-council/issues/>. It is primarily used for:
+
+* Open Council and Committee tasks.
+* Questions for the council.
+* Proposals for new policy and process.
+
+Anyone is welcome to submit issues and to participate in discussion. Alternatively, you may consider bringing up questions and ideas on the [Leadership Council Zulip stream][zulip].
+
+If you want to bring up a security issue, please do not open a public issue on GitHub. Instead, we ask you to refer to Rust's [security policy].
```

Comment:
We should also note this in the issue template.

Reply:
I originally didn't mention it because GitHub automatically creates a link to the security policy from the template selection screen. But it doesn't hurt to mention it again, so I added this in there.